modules/ROOT/pages/deployment/services/s-list/auth-app.adoc

```diff
@@ -0,0 +1,44 @@
+= Auth App Service Configuration
+:toc: right
+
+:service_name: auth-app
+
+:description: The Infinite Scale Auth App service provides authentication for 3rd party apps.
+
+== Introduction
+
+{description}
+
+With the `auth-app` service, you can create tokens that can be used to authenticate 3rd party apps accessing Infinite Scale.
+
+== Default Values
+
+* Auth Basic listens on port 9245 by default.
+
+include::partial$deployment/services/auth-service-family.adoc[]
+
+== Service Startup
+
+Because this service is not started automatically, a manual start needs to be initiated which can be done in several ways. To configure the service usage, an environment variable for the proxy service needs to be set to allow app authentication.
+
+[source,bash]
+----
+OCIS_ADD_RUN_SERVICES=auth-app  # deployment specific. Add the service to the manual startup list, use with binary deployments. Alternatively you can start the service explicitly via the command line.
+
+PROXY_ENABLE_APP_AUTH=true      # mandatory, allow app authentication. In case of a distributed environment, this envvar needs to be set in the the proxy service.
```

Comment:
```suggestion
PROXY_ENABLE_APP_AUTH=true      # mandatory, allow app authentication. In case of a distributed environment, this envvar needs to be set in the proxy service.
```